Gemfile.lock

```diff
@@ -21,14 +34,47 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.13.0)
     rspec-support (3.13.1)
+    rubocop (1.64.1)
```

Comment:
we need to add it as a development dependency. [example](https://github.com/carlosdanielpohlod/rails_tracepoint_stack/blob/main/rails_tracepoint_stack.gemspec#L28)